Include/modsupport.h

```diff
@@ -130,6 +130,11 @@ PyAPI_FUNC(int) PyModule_AddFunctions(PyObject *, PyMethodDef *);
 PyAPI_FUNC(int) PyModule_ExecDef(PyObject *module, PyModuleDef *def);
 #endif
 
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x03070000
```

Comment:
If this should be in the stable API, it needs to be listed in `PC/python3.def` as well. If it does not (and I suspect it does not), remove the second half of this condition.